apps/system/utils/utils_date.c

```diff
@@ -141,7 +141,24 @@ static inline int date_showtime(void)
 	/* Show the current time in the requested format */
 
 	(void)strftime(timbuf, MAX_TIME_STRING, format, &tm);
-	printf("%s\n", timbuf);
+	printf("%s ", timbuf);
+
+	char *zoneinfo = getenv("TZ");
+	if (zoneinfo == NULL) {
+		printf("UTC\n");
+	} else {
+		/* zoneinfo can be like "/etc/zoneinfo/XXX".
+		 * We only need XXX for showing zoneinfo. */
```

Comment:
Below logic might not work if the zoneinfo string is not "/etc/zoneinfo/XXX"
If it is just "XXX" then, it will not be printed because we are checking slash_cnt as 2.

Reply:
Yes, even if it is later changed to a different format, it will be problematic. (xx/xx/etc/zoneinfo/XXX)
May be you can check for zoneinfo and extract the next string after it.

Reply:
Yes Right. I will change. Thanks.